### ISSUE TITLE
Event listeners are not registered

### ISSUE THREAD

**Issue by klawel** [madeindjs/nestjs-graphile-worker#3]
The event listeners in @GraphileWorkerListener() are not registered properly. This is also the case in your sample app. 

Looking for a fix myself, I found that @kiurchv did solve this problem in his fork https://github.com/kiurchv/nestjs-graphile-worker/tree/fix/discovery.

Since you are the owner of the published npm package, could you try to fix the subscription because I really like your wrapper.

**Comment by madeindjs** [comment]
Thanks for the issue.

I open a PR (#4 ) from the branch bellow and I will merge and publish it.

It's weird because it seem to me that `onModuleInit` is a better place for `explore`. Have you a deeper understanding why using `explore` into `constructor` is a better choice ?

**Comment by FlowerWrong** [comment]
Hi! Any update for this issue, bros?

**Comment by madeindjs** [comment]
@FlowerWrong , I reverted the PR who seem to fix this issue (#4). I'm not sure it's a good way to fix it.

**Comment by tayanefernandes** [comment]
Any updates about this issue??

**Comment by madeindjs** [comment]
@tayanefernandes , sadly not... Feel free to publish a PR to fix it, i'll be glad to publish a new version

**Comment by dlpigpen** [comment]
```
@OnWorkerEvent('job:success')
  onJobSuccess({ job }: WorkerEventMap['job:success']) {
    this.logger.debug(`job #${job.id} with payload ${job.payload} finished`);
  }
```
The task is done but the log does not trace out. Could you publish a new PR?

**Comment by madeindjs** [comment]
fixed on [v0.3.1](https://www.npmjs.com/package/nestjs-graphile-worker/v/0.3.1).
